### PR TITLE
Feature/402 GitHub actions

### PR DESCRIPTION
**Pull request by Robin481**

### CODE REVIEW

File: .github/workflows/rails.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Rails
+on: [push, pull_request]
+jobs:
+  Rails-Tests:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11
```
Comment (psunix) [inline .github/workflows/rails.yml:9]: we're using postgres 9.5 in production. 
maybe use same version as in prod here?

File: .github/workflows/rails.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+
+    - name: Setup test database
+      env:
+        RAILS_ENV: test
+        PGHOST: localhost
+        POSTGRES_DB: rails_github_actions_test
```
Comment (psunix) [inline .github/workflows/rails.yml:31]: is there a reason not to go with default values here? e.g. db name skills_test ?

File: .github/workflows/rails.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        POSTGRES_USER: rails_github_actions
+        POSTGRES_PASSWORD: postgres
+      run: |
+        cp config/database.ci.yml config/database.yml
```
Comment (psunix) [inline .github/workflows/rails.yml:35]: is it not possible to go with default datbase.yml here ?

File: .github/workflows/rails.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        bundle exec rake db:create db:migrate
```
Comment (psunix) [inline .github/workflows/rails.yml:36]: is this required? db:migrate is executed when calling rails ci
see lib/tasks/ci.rake

**Review by psunix — COMMENTED** [review]
I don't really know github actions yet ... but added some thoughts

File: .github/workflows/rails.yml (hunk 1 above)
Comment (Robin481) [reply .github/workflows/rails.yml]: Yes totally agree

File: .github/workflows/rails.yml (hunk 4 above)
Comment (Robin481) [reply .github/workflows/rails.yml:36]: Yes, but the CI Task also tries to execute the frontend tests which are not intended to be used in this GitHub Action. We would probably have to split up the rake tasks if we wanted to use the ci rake task. 
I think having the commands in the workflow file makes it simpler to follow anyway.

File: .github/workflows/rails.yml (hunk 3 above)
Comment (Robin481) [reply .github/workflows/rails.yml]: Yes it does indeed work :)

**Pull request merged**